### PR TITLE
test: jestify deploy-cc-from-golang-source-private-data test

### PR DESCRIPTION
**Pull request by awadhana**
Migrated test from Tap to Jest.

File Path:
packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts

This is a PARTIAL resolution to issue #238

Signed-off-by: awadhana <awadhana0825@gmail.com>

### CODE REVIEW

**Review by takeutak — APPROVED** [review]

File: packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts [hunk 1]
```diff
@@ -39,361 +37,356 @@ import { DiscoveryOptions } from "fabric-network";
...
-    signingCredential: {
-      keychainId,
-      keychainRef: keychainEntryKey,
-    },
-  });
+  test(testCase, async () => {
+    const channelId = "mychannel";
+    const channelName = channelId;
+
+    // test.onFailure(async () => {
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts:97]: @awadhana Please delete commented out code that is no longer needed/used in general.

File: packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts [hunk 2]
```diff
@@ -39,361 +37,356 @@ import { DiscoveryOptions } from "fabric-network";
...
+        keychainRef: keychainEntryKey,
+      },
+    });
+
+    expect(getRes).toBeTruthy();
+    expect(getRes.data).toBeTruthy();
+    expect(getRes.data.functionOutput).toBeTruthy();
+    expect(getRes.status > 199).toBe(true);
+    expect(createRes.status < 300).toBe(true);
+    //TODO FIX:
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts:355]: @awadhana Is thi still failing with the errors the comment is saying? If yes we need to do a deep dive.

**Review by petermetz — CHANGES_REQUESTED** [review]
@awadhana Before merging please make sure the PR description and the git commit are updated to not resolve the main jest issue.

File: packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts (hunk 2 above)
Comment (awadhana) [reply packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts:350]: Yes, @petermetz still failing. I'll keep this flagged to go back to in the future.

Comment (awadhana) [reply packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts:350]: image
Actually, it seems to be passing for me now. Let me know if it looks good to you too

Comment (awadhana) [reply packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts:350]: Also, just wanted to notify you @petermetz, everything should be solid now :)

**Review by petermetz — APPROVED** [review]
@awadhana Thank you very much, LGTM!

**Comment by petermetz** [comment]
@awadhana Please rebase onto upstream main + squash the commits into a single one. That will also trigger a retry on the CI (which I think has failed due to some infrastructure outage)

**Pull request merged**